### DISCUSSION TITLE
add ui e2e test

### DISCUSSION THREAD

**Discussion by depapp** [agustinusnathaniel/nextarter-chakra#99] [Ideas]
hi @sozonome 

wdyt about implementing https://playwright.dev/ (ui e2e integration test) into this awesome project?

reference : https://nextjs.org/docs/testing#playwright

**Comment by agustinusnathaniel** [comment]
Hi @depapp ! 👋 

Nice idea! Sorry I just opened this discussion thread

Just added the e2e test through this PR: https://github.com/sozonome/nextarter-chakra/pull/106

Open for suggestions though, just share your ideas / thoughts / PRs 🙇
